src/provisioner/provisioner.py

```diff
@@ -155,46 +170,52 @@ def _get_node_info(cortx_conf: ConfigStore):
 
         # Reinitialize logging with configured log path
         log_path = os.path.join(
-            cortx_conf.get('cortx>common>storage>log'), const.APP_NAME, node_id)
+            config_store.get('cortx>common>storage>log'), const.APP_NAME, node_id)
         log_level = os.getenv('CORTX_PROVISIONER_DEBUG_LEVEL', const.DEFAULT_LOG_LEVEL)
         CortxProvisionerLog.reinitialize(
             const.SERVICE_NAME, log_path, level=log_level)
 
-        if cortx_conf.get(f'node>{node_id}') is None:
+        if config_store.get(f'node>{node_id}') is None:
             raise CortxProvisionerError(
                 errno.EINVAL, f"Node id '{node_id}' not found in cortx config.")
 
-        node_name = cortx_conf.get(f'node>{node_id}>name')
+        node_name = config_store.get(f'node>{node_id}>name')
 
         return node_id, node_name
 
     @staticmethod
-    def _provision_components(cortx_conf: ConfigStore, mp_interfaces: list):
+    def _provision_components(config_store: ConfigStore, mp_interfaces: list, phase_name: str):
         """Invoke Mini Provisioners of cluster components."""
-        node_id, node_name = CortxProvisioner._get_node_info(cortx_conf)
-        components = cortx_conf.get(f'node>{node_id}>components')
+        node_id, node_name = CortxProvisioner._get_node_info(config_store)
+        components = config_store.get(f'node>{node_id}>components')
         if components is None:
             Log.warn(f"No component specified for {node_name} in CORTX config")
         num_components = len(components)
         for interface in mp_interfaces:
             for comp_idx in range(0, num_components):
-                services = cortx_conf.get(
+                services = config_store.get(
                     f'node>{node_id}>components[{comp_idx}]>services')
                 service = 'all' if services is None else ','.join(services)
                 comp_name = components[comp_idx]['name']
+                CortxProvisioner._update_provisioning_info(
+                        config_store, node_id, phase_name, const.PROVISIONING_STATUS.PROGRESS.value)
                 cmd = (
                     f"/opt/seagate/cortx/{comp_name}/bin/{comp_name}_setup {interface}"
-                    f" --config {cortx_conf._conf_url} --services {service}")
+                    f" --config {config_store._conf_url} --services {service}")
```

Comment:
Dont rename cortx_conf to config_store. This change is coming from different patch.

Reply:
Selva's PR is already merge, I have rebased and added this change wherever applicable.

Reply:
Please use cortx_conf instead of config_store. You are reverting his changes.